web-console/future-openshift/cron-jobs/cron-jobs.md

```diff
@@ -0,0 +1,27 @@
+# Cron Jobs
+
+## Create Cron Job
+![](img/cron-job-list.png)
+- Clicking the `Create Cron Job` button launches the creation form.
+
+![](img/create-cron-job.png)
+- Users are prompted to enter a name for the Cron Job, a schedule, and a job template to run.
+- By default, users are presented with a cron expression field and a link to documentation explaining the syntax.
+
+![](img/create-cron-job-2.png)
+- Alternatively, users may select a period which presents the relevant fields for stating when the report should be run
+
+## Suspend Cron Job
+![](img/cron-job-suspend.png)
+- A cron job can be run immediately by clicking the `Run Job` button.
+- Jobs can also be temporarily suspended by selecting `Suspend Jobs` from the actions menu.
```

Comment:
if this is just suspended the current job, should it just be called **Suspend Job**?

Reply:
I believe this suspends all jobs that would execute until the schedule is resumed. Is that right @spadgett?

Reply:
Good idea, I like that!